tests/core/test_hdaccount.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from eth_utils import (
+    ValidationError,
+)
+
+from eth_account import (
+    Account,
+)
+from eth_account.hdaccount import (
+    ETHEREUM_DEFAULT_PATH,
+)
+
+Account.enable_unaudited_hdwallet_features()
+
+
+@pytest.mark.parametrize("mnemonic,account_path,expected_address", [
+    # Ganache
+    # https://github.com/trufflesuite/ganache-core/blob/master/test/accounts.js
+    (
+        "into trim cross then helmet popular suit hammer cart shrug oval student",
+        ETHEREUM_DEFAULT_PATH,
+        "0x604a95C9165Bc95aE016a5299dd7d400dDDBEa9A",
+    ),
+    # Metamask
+    # https://github.com/MetaMask/eth-hd-keyring/blob/master/test/index.js
```

Comment:
Can you update these to be permalinks

https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files